pom.xml

```diff
@@ -63,7 +63,7 @@
 
         <repository>
             <id>jgit-repository</id>
-            <url>http://download.eclipse.org/jgit/maven</url>
+            <url>https://repo.eclipse.org/content/repositories/jgit-releases/</url>
```

Comment:
is this needed?  unclear to me as this is should be proxied by https://repo.jenkins-ci.org/public/  (there is indeed a jgit repo) - but maybe the config in https://repo.jenkins-ci.org/jgit/ is pointing to the now defunct location?)